index.html

```diff
@@ -47,110 +43,115 @@
                 <h1 class="hello_head">Hello, we Welcome you All!!!</h1>
```

Comment:
- [ ] Please add proper styling here by adding gap between main body and header and correct alignment. 
**Clone:** 

![image](https://user-images.githubusercontent.com/58504655/166084402-e24a40ae-f8ac-4fef-ac54-4a2129cb39c6.png)

**Original**: 
![image](https://user-images.githubusercontent.com/58504655/166084415-bb1d5dec-32c0-4111-95f1-cf9c8ffbfecf.png)

Reply:
@arslanbisharat Thank you for the feedback.
the above-requested changes have been implemented and updated.